test/README.md

```diff
@@ -0,0 +1,40 @@
+Full-Stack tests may be written for any given bug, they live in the test directory`.
+
+A test must be named using the issueXXXX-thing-fails-for-thing convention. A bash script ( with executable flag set ) and folder should be created with this name.
+
+The script may execute arbitrary commands designed to replicate the bug in question, it may use the folder of the same name in any way to support recreating this bug.
+Scripts have a number of special variables they may access:
+
+```
+$DC    The D compiler
+$DUB    The built dub executable
+$CURR_DIR    The current directory
+$HOME    The home directory of the user
```

Comment:
That's just inherited from the env though, so not sure it's worth mentioning.
Also no test script should use `$HOME`, it should be self-contained!